conf/application.staging.conf

```diff
@@ -30,6 +30,8 @@ akka.actor.warn-about-java-serializer-usage="off"
 
 play.filters.headers.frameOptions = "SAMEORIGIN"
 
+play.http.forwarded.trustedProxies=[${ALB_IP_A}, ${ALB_IP_B}]
```

Comment:
This made me think: do you think it's worth moving most of the settings to a base config file like we do in the API? https://github.com/nationalarchives/tdr-consignment-api/blob/master/src/main/resources/application.intg.conf#L1